gene2phenotype_project/gene2phenotype_app/urls.py

```diff
@@ -2,95 +2,204 @@
 from rest_framework.urlpatterns import format_suffix_patterns
 from gene2phenotype_app import views
 
+
 def perform_create(self, serializer):
     serializer.save(owner=self.request.user)
 
+
 # specify URL Path for rest_framework
 urlpatterns = [
-    path('', views.ListEndpoints, name="list_endpoints"),
-    path('panels/', views.PanelList.as_view(), name="list_panels"),
-    path('panel/<str:name>/', views.PanelDetail.as_view(), name="panel_details"),
-    path('panel/<str:name>/summary/', views.PanelRecordsSummary.as_view(), name="panel_summary"),
-    path('panel/<str:name>/download/', views.PanelDownload, name="panel_download"),
-    path('users/', views.UserList.as_view(), name="list_users"),
-    path('user/panels/', views.UserPanels.as_view(), name="user_panels"),
-    path('attribs/', views.AttribTypeList.as_view(), name="list_attrib_type"),
-    path('attribs/description', views.AttribTypeDescriptionList.as_view(), name="description_attrib_type"),
-    path('attrib/<str:code>/', views.AttribList.as_view(), name="list_attribs_by_type"),
-    path('molecular_mechanisms/', views.ListMolecularMechanisms.as_view(), name="list_mechanisms"),
-    path('ontology_terms/variant_types/', views.VariantTypesList.as_view(), name="list_variant_types"),
-    path('gene/<str:name>/', views.LocusGene.as_view(), name="locus_gene"),
-    path('gene/<str:name>/summary/', views.LocusGeneSummary.as_view(), name="locus_gene_summary"),
-    path('gene/<str:name>/function/', views.GeneFunction.as_view(), name="locus_gene_function"),
-    path('gene/<str:name>/disease/', views.GeneDiseaseView.as_view(), name="locus_gene_disease"),
-    path('disease/<path:id>/summary/', views.DiseaseSummary.as_view(), name="disease_summary"),
-    path('disease/<path:id>/', views.DiseaseDetail.as_view(), name="disease_details"),
-    path('publication/<str:pmids>/', views.PublicationDetail, name="publication_details"),
-    path('phenotype/<str:hpo_list>/', views.PhenotypeDetail, name="phenotype_details"),
-    path('lgd/<str:stable_id>/', views.LocusGenotypeDiseaseDetail.as_view(), name="lgd"),
-    path('search/', views.SearchView.as_view(), name="search"),
-
+    path("", views.ListEndpoints, name="list_endpoints"),
+    path("panels/", views.PanelList.as_view(), name="list_panels"),
+    path("panel/<str:name>/", views.PanelDetail.as_view(), name="panel_details"),
+    path(
+        "panel/<str:name>/summary/",
+        views.PanelRecordsSummary.as_view(),
+        name="panel_summary",
+    ),
+    path("panel/<str:name>/download/", views.PanelDownload, name="panel_download"),
```

Comment:
Updating the URLs format is a good idea, but it gets confusing when some endpoints are represented in one line and other are splitted. It's better to apply the same format to all of them.